pyp2rpm/templates/epel6.spec

```diff
@@ -143,7 +143,13 @@ popd
 {{ '%{python2_sitelib}'|sitedir_for_python_version(pv) }}/{{ underscored_or_pypi(data.name, data.underscored_name) }}-%{version}-py?.?.egg-info
 {%- endif %}
 {% endcall %}
-
+{%- if data.sphinx_dir %}
+%files -n {{ data.pkg_name|macroed_pkg_name(data.srcname)|name_for_python_version(pv, True) }}-doc
+%doc html
+{%- if data.doc_license %}
+%license {{data.doc_license|join(' ')}}
```

Comment:
Yes it is available in EPEL, [documentation](https://fedoraproject.org/wiki/EPEL:Packaging#The_.25license_tag) says it is mapped to %doc.  This macro is present in some of the specfiles for EPEL [python-pip](http://pkgs.fedoraproject.org/cgit/rpms/python-pip.git/tree/python-pip.spec?h=el6#n207) for example.